xml/upgrade-2.10.xsl

```diff
@@ -0,0 +1,10 @@
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+<xsl:output method="xml" encoding="UTF-8" indent="yes"/>
+
+<xsl:template match="@*|*">
+  <xsl:copy>
+    <xsl:apply-templates select="@*|node()"/>
+  </xsl:copy>
+</xsl:template>
+
+</xsl:stylesheet>
```

Comment:
Is it the opportunity to drop score-attribute{,-mangled}:
https://github.com/ClusterLabs/pacemaker/pull/1091
?

Reply:
Yes, definitely :) I'm planning a big update of the XSL separately, then I'll probably pull 2.0 into the master branch to make sure they don't get too out-of-sync